howto/bug-report/landing.md

```diff
@@ -0,0 +1,36 @@
+*Applies to: Anbox Cloud, Anbox Cloud Appliance since 1.16.0*
+
+In some cases, you might encounter issues with Anbox Cloud that you are not able
+to troubleshoot yourself, or that may be bugs in the software itself.
+
+Before applying the steps in this guide, please check the instructions for
+commonly encountered problems at [How to troubleshoot Anbox
+Cloud](https://discourse.ubuntu.com/t/how-to-troubleshoot-anbox-cloud/17837)
+first.
+
+## Build a bug report
+
+Anbox containers come preinstalled with the `anbox-bug-report` utility. On a
+running container, use `amc exec` to build the report:
+```
+amc exec <container_id> -- anbox-bug-report
+```
+
+After a few seconds, `anbox-bug-report` builds a zip archive and prints its
+absolute path.
+
+## Retrieve a bug report
+
+The generated bug report can be retrieved using an LXD client:
+```
+lxc file pull <LXD_server>:<container_id>/<log_path> <target_path>
```

Comment:
The `container_id` here has to be prefixed with `ams-` if people want to pull the file by using the
`lxc file pull` command.

To me, using `lxc file pull` is sub-optimal here, because if people deploy a regular deployment(not appliance), this means people have to go to one LXD node, and run the above command from there. I may consider

- if the container remains running, using `ams exec` command may be more suitable here. E.g.

```
amc exec <container_id> -- sh -c "cat <log_path>" | cat - > <target_path>
``` 

- if the container is stopped E.g. ran into an error because of the watchdog. using the `lxc file pull` instead.

Reply:
> The container_id here has to be prefixed with ams- if people want to pull the file by using the lxc file pull command.

Will update.

> To me, using lxc file pull is sub-optimal here, because if people deploy a regular deployment(not appliance), this means people have to go to one LXD node, and run the above command from there.

Do they have to find the IP address of the individual LXD node?

> if the container is stopped E.g. ran into an error because of the watchdog. using the lxc file pull instead.

If the container stopped, there should be no way to run `anbox-bug-report` then, so you can just `amc exec` in that case.

Reply:
An important benefit of using `cat` instead of `lxc file pull` is that it becomes an easy one-liner:
```bash
amc exec <container_id> -- bash -c 'cat "$(anbox-bug-report)"' > <target_file>
```

Reply:
We agreed on using `amc exec` for now.